plugins/cost-insights/src/client.ts

```diff
@@ -101,7 +102,10 @@ export class ExampleCostInsightsClient implements CostInsightsApi {
         trendline: trendlineOf(aggregation),
         // Optional field on Cost which needs to be supplied in order to see
```

Comment:
nitpicky, but we can probably generalize this rather than saying `...field on Cost which needs to be supplied to see the product breakdown view...`